webpack/webpack.config.cjs

```diff
@@ -175,10 +175,6 @@ module.exports = function(env, argv) {
         },
       },
     },
-    cache: {
-      type: dev ? 'memory' : 'filesystem',
-      name: 'cactbot',
-    },
```

Comment:
This seems unrelated? If you wanted to remove this, can you send a different PR?

Reply:
Oh yes, I'll remove that.